include/openssl/cmp.h

```diff
@@ -233,8 +236,108 @@ int OSSL_CMP_ITAV_push0_stack_item(STACK_OF(OSSL_CMP_ITAV) **itav_sk_p,
                                    OSSL_CMP_ITAV *itav);
 void OSSL_CMP_ITAV_free(OSSL_CMP_ITAV *itav);
 void OSSL_CMP_MSG_free(OSSL_CMP_MSG *msg);
-void OSSL_CMP_PKISI_free(OSSL_CMP_PKISI *si);
-DECLARE_ASN1_DUP_FUNCTION(OSSL_CMP_MSG)
+void OSSL_CMP_PKISI_free(OSSL_CMP_PKISI *si); /* not really needed */
+DECLARE_ASN1_DUP_FUNCTION(OSSL_CMP_MSG) /* not really needed */
```

Comment:
not needed ?

Reply:
The comments were meant as follows: I found that these two functions are so far only used internally, so are not strictly required in the public API. 
Yet since this section of `cmp.h` has already been approved and merged as part of chunk 3 and the two declarations do not hurt, I'd prefer not to move them to `cmp_int.h` (which also would require changing `libcrypto.num` and the docs).

Reply:
Okay, then change that to "TODO: move to internal header before 3.0.0 release"
or so?

Reply:
Or shall we do the move right away, since I see little benefit in just deferring this?

Reply:
Fixed: Moved to cmp_int.h